Helping README readability and fixing minor typo/grammar issues (#10590)
WordPress/gutenberg: README.md

## Getting started
- **Download:** If you want to use the latest release with your WordPress, <a href="https://wordpress.org/plugins/gutenberg/">download the latest release from the WordPress.org plugins repository</a>. 
- **Discuss:** Conversations and discussions take place in #core-editor on <a href="https://make.wordpress.org/chat/">the core WordPress Slack</a>.
- **Contribute:** Development happens here. Get started by reading the <a href="https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md">CONTRIBUTING</a> guidelines.
- **Learn:** Discover more about the project <a href="https://wordpress.org/gutenberg">here</a>.

**Gutenberg is more than an editor.** While the project is currently focused on building the new editor for WordPress, it doesn't end there. This lays the groundwork for a new model for WordPress Core that will ultimately impact the entire publishing experience of the platform.

## Editing focus

> *The editor will create a new page- and post-building experience that makes writing rich posts effortless, and has “blocks” to make it easy what today might take shortcodes, custom HTML, or “mystery meat” embed discovery.*
>
> — Matt Mullenweg

> One thing that sets WordPress apart from other systems is that it allows you to create a post layout as rich as you can imagine—but only if you know HTML and CSS and build your own custom theme. By thinking of the editor as a tool that can let you both write rich posts and create beautiful layouts, we can transform WordPress into something users _love_, as opposed something they pick because it happens to be what everyone else uses.

Reviewer: It was in the original text, but 'as opposed something' seems like a mistake. Shouldn't it be 'as opposed to something', or 'instead of something'.

Reviewer 2: Hello @talldan, thanks for the copy review ping. Here's a suggestion to add a couple of words that seem to be missing and streamline the text just slightly. 

```suggestion
One thing that sets WordPress apart is that it allows you to create a post layout that's as rich as you can imagine—but only if you know HTML and CSS and can build your own custom theme. By thinking of the editor as a tool that allows you to write rich posts **and** create beautiful layouts, we can transform WordPress into something users _love_, as opposed to something they choose because it happens to be what everyone else uses.
```

> **Gutenberg is a new way forward.** It looks at the editor as more than a content field, revisiting a layout that has been largely unchanged for almost a decade. This project allows us to holistically design a modern editing experience, and build a foundation for things to come.

Reviewer: I've seen a bit of discussion recently about using 'us' or 'we', which doesn't read very inclusively. It could be something to change here.

Using 'the WordPress community' as a replacement might be an idea.

Reviewer 2: Looks great! I have a minor quibble with the revision in that stylistically, there isn't a need for a comma after "experience" as the word "and" acts as a natural pause.

Here's why we're looking at the whole editing screen, as opposed to just the content field:

> 1. **The block unifies multiple interfaces.** If we add that on top of the existing interface, it would _add_ complexity, as opposed to removing it.

Reviewer: 'Introducing blocks as part of the existing editor would add complexity'

Reviewer: Also noting we use 'as opposed to' here 😄

Reviewer 2: For clarity, what are we referring to when we say, "that" in the first sentence [If we add "that"...]? I found that sentence confusing.

3. **Better interface usability.** When singular block interface takes center stage, it demonstrates a clear path forward for developers to create premium blocks, superior to both shortcodes and widgets.
4. **A fresh look at content creation.** Considering the whole interface lays a solid foundation for the next focus, full site customization.
5. **Modern tooling.** Looking at the full editor screen also gives us the opportunity to drastically modernize the foundation, and take steps towards a more fluid and JavaScript-powered future that fully leverages the WordPress REST API.

![Writing in Gutenberg 1.6](https://make.wordpress.org/core/files/2017/10/gutenberg-typing-1_6.gif)

## Blocks

Blocks are the unifying evolution of what is now covered, in different ways, by shortcodes, embeds, widgets, post formats, custom post types, theme options, meta-boxes, and other formatting elements. They embrace the breadth of functionality WordPress is capable of, with the clarity of a consistent user experience.

> Imagine a custom `employee` block that a client can drag onto an `About` page to automatically display a picture, name, and bio of all the employees. Imagine a whole universe of plugins just as flexible, all extending WordPress in the same way. Imagine simplified menus and widgets. Users who can instantly understand and use WordPress—and 90% of plugins. This will allow you to easily compose beautiful posts like <a href="http://moc.co/sandbox/example-post/">this example</a>.

Reviewer: I think it's worth removing the definitive article here, since these employees haven't been previously established in the writing.
'display a picture, name, and bio of all the employees.' -> 'display a picture, name, and bio of employees.'

Reviewer 2: Just a couple small suggestions here. 

```suggestion
Imagine a custom `employee` block that a client can drag onto an `About` page to automatically display a picture, name, and bio. Imagine a whole universe of plugins just as customizable, all extending WordPress in the same way. Imagine simplified menus and widgets. Imagine a scenario where Users can instantly understand and use WordPress—and 90% of plugins. This will allow you to easily compose beautiful posts like <a href="http://moc.co/sandbox/example-post/">this example</a>.
```

## The stages of Gutenberg

Gutenberg has three planned stages.
1) **The first, aimed for inclusion in WordPress 5.0, focuses on the post editing experience** and the implementation of blocks. This initial phase focuses on a content-first approach. The use of blocks, as detailed above, allows you to focus on how your content will look without the distraction of other configuration options. This ultimately will help all users present their content in a way that is engaging, direct, and visual. These foundational elements will pave the way forward.

> 2) Planned for 2019, **The second stage focuses on overhauling The Customizer** and page templates.

Reviewer: It wonder if it might be worth asking for a brief description here from the leads? 'overhauling' is quite an extreme word, and it also implies the customizer lives on, whereas really we could be imagining and exploring entirely new ways to theme and customize.

Reviewer 2: Nothing to suggest here.

> 3) Ultimately, **full site customization** will be possible.

Reviewer: I don't know if we want to talk about the third phase and onwards yet, so it might be worth not mentioning it.

Reviewer 2: Nothing to suggest here.
